app/controllers/api/auth/stories_controller.rb

```diff
@@ -45,11 +46,37 @@ def filtered(resources)
   end
 
   def resources_base
-    # If there is a network_id specified, use that network
+    # If there is a network_id specified, use that network.
+    # The authz is performed in check_user_network().
     @stories ||= if params[:network_id]
       super.published
     else
       authorization.token_auth_stories
     end
   end
+
+  def search
+    # same logic as resources_base above
+    @stories ||= if params[:network_id]
```

Comment:
Something is slightly wonky when searching with a network_id.  Digging around in the SQL a bit, the difference seems to be the `Story.network_visible` scope is applied by `api/v1/networks/123/stories` but not `api/v1/stories/search?q=network_id:123`.

Reply:
Another "funny" discrepancy ... `api/v1/accounts/123/stories` is actually the equivalent of `account.public_stories`, not `account.stories.published`.  The `public_stories` includes some networks logic as well.

And now that i'm searching for that, it appears that networks/series/accounts filters all use "public" instead of "published".